pkg/kubemark/controller.go

```diff
@@ -247,8 +248,7 @@ func (kubemarkController *KubemarkController) RemoveNodeFromNodeGroup(nodeGroup
 	policy := metav1.DeletePropagationForeground
 	var err error
 	for i := 0; i < numRetries; i++ {
-		err = kubemarkController.externalCluster.client.CoreV1().ReplicationControllers(namespaceKubemark).Delete(
-			pod.ObjectMeta.Labels["name"],
```

Comment:
if this was a test to see if I'd notice any non-mechanical changes, I noticed :)

Reply:
I'm very impressed and refuse to fix it. I originally attempted to reprint the full ast of modified files but ran into unanchored comments jumping around https://github.com/golang/go/issues/20744. Rather than figuring out the position stuff, I ended up just printing only the call expression and splicing the original source.

Reply:
> I'm very impressed and refuse to fix it

oh, I don't care